pixorama_flutter/lib/main.dart

```diff
@@ -1,24 +1,22 @@
 import 'package:made_with_serverpod/made_with_serverpod.dart';
 import 'package:pixorama_client/pixorama_client.dart';
 import 'package:flutter/material.dart';
-import 'package:pixorama_flutter/src/pixorama/pixorama.dart';
+import 'package:pixorama_flutter/src/pixorama.dart';
 import 'package:serverpod_flutter/serverpod_flutter.dart';
 import 'package:url_strategy/url_strategy.dart';
 
-// Change this to use the live server or to connect to a local server.
-var client = Client('https://api.pixorama.live/');
-// var client = Client('http://localhost:8080/');
+// ignore: constant_identifier_names
```

Comment:
Never ignore warnings, unless there is a very good reason for doing so. If we want to support the feature to switch between local and production this should be done with a build time constant passed to the `flutter` command when starting/building the app.